src/magister.js

```diff
@@ -426,8 +426,21 @@ class Magister {
 			self.profileInfo = new ProfileInfo(self, accountData.Persoon)
 			self._privileges = new Privileges(self, accountData.Groep[0].Privileges)
 
+			// REVIEW: Maybe pass this as an argument to the ProfileInfo constructor?
+			// If someone has this privilege then they are a parent
+			self.profileInfo.isChild = !(await self._privileges.can("kinderen", "read"))
+
 			self._personUrl = `${schoolUrl}/api/personen/${id}`
 			self._pupilUrl = `${schoolUrl}/api/leerlingen/${id}`
+
+			// REVIEW: Maybe pass this as an argument to the ProfileInfo constructor?
+			// If we are a child then check if our parents can see our information
+			if (self.profileInfo.isChild) {
+				self.profileInfo.isVisibleForParent =
```

Comment:
I think we can just remove this property from the `ProfileInfo` class completely, and just make it a method which does the `autorisatie` check.

Reply:
(an async function which returns a `bool`, and always false if `!isParent`)